src/pages/documentation/adobe-analytics/tabs/index.md

```diff
@@ -1,11 +1,11 @@
-<Variant platform="android" task="add" repeat="6"/>
+<Variant platform="android" task="add" repeat="8"/>
 
-#### Java
 
 1. Add the [Mobile Core](../mobile-core/index.md) and Analytics extensions to your project using the app's Gradle file.
 
 ```java
 implementation 'com.adobe.marketing.mobile:core:2.x.x'
+implementation 'com.adobe.marketing.mobile:identity:2.x.x'
 implementation 'com.adobe.marketing.mobile:analytics:2.x.x'
 ```
```

Comment:
for consistency can we use the 2.+ notation here and include a warning hint above on how to lock in gradle dependency versions (link to manage-gradle-dependencies)